sklearn/__init__.py

```diff
@@ -47,6 +48,13 @@
 __version__ = '0.21.dev0'
 
 
+# On OSX, we can get a runtime error due to multiple OpenMP libraries loaded
+# simultaneously. This can happen for instance when calling BLAS inside a
+# prange. Setting the following environment variable allows multiple OpenMP
+# libraries.
```

Comment:
I would add: in sections of the code where nested OpenMP loops can happen, we manually take care of potential over-subcription performance problems by dynamically reconfiguring the inner OpenMP runtime to temporarily disable it while under the scope of the outer OpenMP parallel section.